### PR TITLE
Add start time to server started message

### PR DESCRIPTION
**Pull request by tjwatson**
Startup time is becoming important for us to monitor. This PR adds to the server started message the time taken to start. The message now looks like this:

`[AUDIT   ] CWWKF0011I: The server defaultServer is ready to run a smarter planet. It started in 0.933 seconds.`

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
#build

File: dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops [hunk 1]
```diff
@@ -70,7 +70,7 @@ BUNDLE_MATCH_WARNING=CWWKF0010W: More than one bundle matched the specified filt
 BUNDLE_MATCH_WARNING.explanation=A feature definition element matched more than one bundle. This could be because the filter is too general, or because there are old bundles in the image that match the filter.
 BUNDLE_MATCH_WARNING.useraction=Make sure that the filter is sufficiently specific, and that a clean install image is being used.
 
-SERVER_STARTED=CWWKF0011I: The server {0} is ready to run a smarter planet.
+SERVER_STARTED=CWWKF0011I: The server {0} is ready to run a smarter planet. It started in {1} seconds.
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops]: Change "The server {0} is ready to run a smarter planet."
To "The {0} server is ready to run a smarter planet."

Change "It started in {1} seconds."
To "The {0} server started in {1} seconds."

File: dev/com.ibm.ws.kernel.feature/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops [hunk 2]
```diff
@@ -70,7 +70,7 @@ BUNDLE_MATCH_WARNING=CWWKF0010W: More than one bundle matched the specified filt
 BUNDLE_MATCH_WARNING.explanation=A feature definition element matched more than one bundle. This could be because the filter is too general, or because there are old bundles in the image that match the filter.
 BUNDLE_MATCH_WARNING.useraction=Make sure that the filter is sufficiently specific, and that a clean install image is being used.
 
-SERVER_STARTED=CWWKF0011I: The server {0} is ready to run a smarter planet.
+SERVER_STARTED=CWWKF0011I: The server {0} is ready to run a smarter planet. It started in {1} seconds.
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.feature/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops]: Change "The server {0} is ready to run a smarter planet."
To "The {0} server is ready to run a smarter planet."

Change "It started in {1} seconds."
To "The {0} server started in {1} seconds."

Comment (tjwatson) [reply dev/com.ibm.ws.kernel.feature/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops]: done

**Comment by tjwatson** [comment]
#build

**Comment by dmuelle** [comment]
Suggested edits were made to the files- applying the ID Reviewed Tag

**Comment by skasund** [comment]
L2 message review completed

**Pull request merged**